### ISSUE TITLE
HHM cropping unstable - flanking regions important

### ISSUE THREAD

**Issue by peterjc** [peterjc/thapbi-pict#192]
Noticed during import, but has the potential to affect processing FASTQ input too. This example only alters the right (end) trimming.

Test case:

```python
from Bio import SeqIO
from Bio._py3k import StringIO

original = """                                                                                                                                                                                                               
>HQ643763.1 Pythium rostratifingens voucher Lev3023 18S ribosomal RNA gene, partial sequence; internal transcribed spacer 1, 5.8S ribosomal RNA gene, and internal transcribed spacer 2, complete sequence; and 28S ribosoma\
l RNA gene, partial sequence.                                                                                                                                                                                                
CACACCTAAAAACTATCCACGTGAACCGTTAAGCAAACAAGTTAAGGCAGGCGCGATTGGTGGTGCGTCTGGGAAGCGCTTGTCGATATTCGATCGGATCGGATATTGTCGGGCGTCGTCCTGGACACGCTGTCGATCGGAGTCGGCTAAACGAAGGTCGGGCGTTCGCTATCGGAGCGATGTGCGCGTTGTCGCACGTTGCTGCAATGGCTCGAGCAAG\
CGGCTGATTTATGTCTTTCAAACCATACGTGACGTACTGATTATACTGTGAGGACGAAAGTCCTTGCTTTTACTAGATAACAACTTTCAGCAGTGGATGTCTAGGCTCGCACATCGATGAAGAACGCTGCGAACTGCGATACGTAATGCGAATTGMAGAATTCAGTGAGTCATCGAAATTTTGAACGCATATTGCACTTCCGGGTTATGCCTGGAAGTAT\
GTCTGTATCAGTGTCCGTACATTAAACTTGCCTCTCTTCGTCGGTGTAGTCCGGCTTGGAGAAGGAGCAGAGGTGAGTGTCTCGCGCCATGCTGKTGATCTATCTTCRGRTAGATGACGAGAGTGCACG                                                                                            
"""

record = SeqIO.read(StringIO(original), "fasta")
species = "Pythium rostratifingens"
for i in reversed(range(100, len(record) + 1)):
    print(">%s.c%i %s (cropped %i)\n%s" % (record.id, i, species, i, record.seq[:i]))
```

Usage:

```
$ python HQ643763_crop.py > HQ643763_crop.fasta && rm -rf HQ643763_crop.sqlite && thapbi_pict ncbi-import -d HQ643763_crop.sqlite -i HQ643763_crop.fasta -x && thapbi_pict dump -d HQ643763_crop.sqlite | cut -f 1-5 | sort
File HQ643763_crop.fasta had 470 sequences. Found 155 with ITS1, of which 155 accepted.
Of 155 potential entries, loaded 155 entries, 0 failed parsing.
Wrote 155 txt format entries
#Identifier	Genus	Species	TaxID	ITS1-MD5
HQ643763.1.c259	Pythium	rostratifingens		a6582b93840b70e21839ea4fdf9a039c
HQ643763.1.c263	Pythium	rostratifingens		b9e9912f94376888fa868b2f473d5f67
HQ643763.1.c265	Pythium	rostratifingens		140bb03b6275c2140d664547b1f364d6
HQ643763.1.c266	Pythium	rostratifingens		539baa719e6bfd45fecf16440371f2e1
HQ643763.1.c267	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c268	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c269	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c270	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c271	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c272	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c273	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c274	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c275	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c276	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c277	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c278	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c279	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c280	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c281	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c282	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c283	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c284	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c285	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c286	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c287	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c288	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c291	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c292	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c293	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c294	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c295	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c296	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c298	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c299	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c349	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c350	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c351	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c352	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c357	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c358	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c359	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c360	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c361	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c362	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c363	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c364	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c369	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c370	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c371	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c374	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c375	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c376	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c377	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c378	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c379	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c380	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c381	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c382	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c383	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c384	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c385	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c386	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c411	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c412	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c413	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c414	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c415	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c416	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c417	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c418	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c419	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c420	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c421	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c422	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c425	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c426	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c427	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c428	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c429	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c430	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c431	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c432	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c433	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c434	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c435	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c436	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c437	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c438	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c439	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c440	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c441	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c442	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c443	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c444	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c445	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c446	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c447	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c448	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c449	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c450	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c451	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c452	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c453	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c455	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c456	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c457	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c458	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c459	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c460	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c461	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c462	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c463	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c464	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c465	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c466	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c467	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c468	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c469	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c518	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c522	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c523	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c524	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c525	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c526	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c527	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c528	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c529	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c530	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c531	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c532	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c533	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c534	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c535	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c536	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c537	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c538	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c539	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c540	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c541	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c542	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c543	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c544	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c545	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c546	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c547	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c548	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c549	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c550	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c551	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c552	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c553	Pythium	rostratifingens		bd687591ef17892d295dd3f639e40736
HQ643763.1.c566	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c567	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c568	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
HQ643763.1.c569	Pythium	rostratifingens		9f913833ebf1e299b6dc7b71efacb385
```

Here 5ef469138a501f7443029c604fd49b3d is the longest entry:

```
$  thapbi_pict dump -d HQ643763_crop.sqlite -m
#MD5	Species	Sequence
b9e9912f94376888fa868b2f473d5f67	Pythium rostratifingens	CACACCTAAAAACTATCCACGTGAACCGTTAAGCAAACAAGTTAAGGCAGGCGCGATTGGTGGTGCGTCTGGGAAGCGCTTGTCGATATTCGATCGGATCGGATATT
bd687591ef17892d295dd3f639e40736	Pythium rostratifingens	CGATCGGAGTCGGCTAAACGAAGGTCGGGCGTTCGCTATCGGAGCGATGTGCGCGTTGTCGCACGTTGCTGCAATGGCTCGAGCAAGCGGCTGATTTATGTCTTTCAAACCATACGTGACGTACTGATTATACT
9f913833ebf1e299b6dc7b71efacb385	Pythium rostratifingens	TCGATCGGAGTCGGCTAAACGAAGGTCGGGCGTTCGCTATCGGAGCGATGTGCGCGTTGTCGCACGTTGCTGCAATGGCTCGAGCAAGCGGCTGATTTATGTCTTTCAAACCATACGTGACGTACTGATTATACT
a6582b93840b70e21839ea4fdf9a039c	Pythium rostratifingens	TCGGGCGTCGTCCTGGACACGCTGTCGATCGGAGTCGGCTAAACGAAGGTCGGGCGTTCGCTATCGGAGCGATGTGCGCGTTGTCGCACGTTGCTGCAATGGCTCGAGCAAGCGGCTGATTTATGTCTTTCAAACCATACG
140bb03b6275c2140d664547b1f364d6	Pythium rostratifingens	TCGGGCGTCGTCCTGGACACGCTGTCGATCGGAGTCGGCTAAACGAAGGTCGGGCGTTCGCTATCGGAGCGATGTGCGCGTTGTCGCACGTTGCTGCAATGGCTCGAGCAAGCGGCTGATTTATGTCTTTCAAACCATACGTGACGTACTGATTATA
539baa719e6bfd45fecf16440371f2e1	Pythium rostratifingens	TCGGGCGTCGTCCTGGACACGCTGTCGATCGGAGTCGGCTAAACGAAGGTCGGGCGTTCGCTATCGGAGCGATGTGCGCGTTGTCGCACGTTGCTGCAATGGCTCGAGCAAGCGGCTGATTTATGTCTTTCAAACCATACGTGACGTACTGATTATAC
5ef469138a501f7443029c604fd49b3d	Pythium rostratifingens	TCGGGCGTCGTCCTGGACACGCTGTCGATCGGAGTCGGCTAAACGAAGGTCGGGCGTTCGCTATCGGAGCGATGTGCGCGTTGTCGCACGTTGCTGCAATGGCTCGAGCAAGCGGCTGATTTATGTCTTTCAAACCATACGTGACGTACTGATTATACT
Wrote 7 txt format entries
```

The original HQ643763.1 is 569bp, and gives a match to 9f913833ebf1e299b6dc7b71efacb385 which is relatively truncated. The other near full length queries behave similarly.

The shortest sequence accepted was cropped down from 569bp to 259bp, so you might expect everything in between to be accepted (i.e. 311 sequences), but only about half were (155 accepted).

When the query length is 300 to 348bp, we get no match - yet this is inside the window where we see the longest matches!

We only see the longest match 5ef469138a501f7443029c604fd49b3d when the query length is between 267 and 371bp - _but not for all those lengths_.

```
HQ643763.1.c267	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c269	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c270	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c271	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c272	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c273	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c274	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c275	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c276	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c277	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c278	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c279	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c286	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c287	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c288	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c291	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c292	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c293	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c294	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c295	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c296	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c298	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c299	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c349	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c350	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c351	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c352	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c369	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c370	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
HQ643763.1.c371	Pythium	rostratifingens		5ef469138a501f7443029c604fd49b3d
```

Question: Is this pathological behaviour only seen with non-*Phytophthora* (possible as the HMM is built from *Phytophthora* only)? If so, need to be clearer about the limits.

Option A: Determine how far outside the *Phytophthora* this HMM can be reliably used, and limit our default DB entries accordingly.

Option B: Build a new HMM, which we will likely need to do anyway if we wish to expand much beyond the *Phytophthora*.

Option C: Stop using HMM cropping (just use for accept/reject). For processing FASTQ files, just use the primer based cutting. For the database, the single isolate controls and manually curated (hand trimmed) sequences should be OK. However, the current mass import of NCBI ITS1 results (currently recorded only to genus level) would have to be modified - can we apply the primer trimming to longer public sequences?

Next step: Screen some NCBI downloads for this kind of behaviour to assess how widespread it is within each genus.

**Comment by peterjc** [comment]
Option D: Apply *in silico* primer trimming as part of DB import, but do not require both primers be found (as with many sequences they will be only the region of interest). This should give a far narrower window of sequence lengths to hmmscan, and thus indirectly reduce this flanking effect.

**Comment by peterjc** [comment]
Option D seems very promising...

**Comment by peterjc** [comment]
Work in v0.4.4 allows running with ``--hmm ''`` to disable the HMM cropping, motivation was trying other markers where did not yet have a HMM.

**Comment by peterjc** [comment]
Closed via #193, option D, apply primer trimming on import to dramatically reduce the variation in flanking regions.
